public/data/Obito9596.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I don't think you should add a profile for others. 🤔

Reply:
this is my profile earlier, i changed the name to YashKumarShrivas right now, that's why along with new pr the old one also added. 
Obito9596-->YashkShrivas4491
sorry for the inconvenience caused by me.